src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -7686,30 +7803,68 @@ public class MonologAboutBookCallableCallableStreamingClientProg {
 
 // FIXME: Insert here code to prepare the request fields, make the call, process the response.
 
+package com.google.gcloud.pubsub.v1;
```

Comment:
Why is this in pubsub?

Reply:
It's defined [here](https://github.com/googleapis/gapic-generator/blob/b4c1fb4c4e24f9cbb32c9c91894c7de17d6ca47b/src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml#L5).
Shall we change the package above to `com.google.cloud.example.library.v1`?
And therefore, the samples will be in the package `com.google.cloud.example.example.library.v1.snippets`, as to be consistent with examples in `google-cloud-java`?